content/docs/data_registry/schema.md

```diff
@@ -1,6 +1,55 @@
 ---
-title: "FAIR Data Registry Schema"
-weight: 1
+title: "Schema Description"
+weight: 2
 ---
 
-Description of the schema
\ No newline at end of file
+The FAIR data registry schema has been designed to address the different requirements around the information needed on different research objects associated with an epidemiological pipeline.
+
+## Main entities
+
+The main entities represented in the schema are around the data and the software for epidemiological models:
+- **DataProduct**: A dataset that is used by or is generated by a model.
+- **ExternalObject**: An external data object, i.e. one that has comes from somewhere other than being generated as part of the modelling pipeline.
+- **CodeRun**: A code run along with its associated, code repo, configuration, input and outputs.
+- **Object**: Core traceability object used to represent any data object such `DataProduct`, `CodeRepoRelease`, etc. 
+### Data representation
+
+For the data representation, the schema also supports categorising datasets by using:
+- **Namespace**: A namespace is a way to group `DataProduct`s following some specific categorisation criteria, such as what is the organisation producing the data product.
+
+
+
+![data products](/docs/data_registry/FAIRdataregistry-dataproduct.jpg)
+
+### Models representation
+
+![code runs](/docs/data_registry/FAIRdataregistry-coderun.jpg)
```

Comment:
I might add `Object`s off `ObjectComponent`s here for clarity, since we don't mention components much.